README.md

```diff
@@ -134,6 +136,16 @@ You'll need to run the producer every minute to create jobs the consumer can pro
 * * * * * /usr/bin/env php path/to/datatidy/bin/console datatidy:data-flow:produce-jobs > path/to/output.file
 ```
 
+#### Handling long running jobs
+
+Sometimes and for different reasons a job may run for a long time. And because jobs only can be created if there is no other active jobs for a DataFlow, you need to set those jobs in a non-active state.
+To help you doing this there is a command available for doing this. It uses the JOB_TIMEOUT_THRESHOLD environment variable to cancel all jobs that have been running without finishing after the threshold is reached.
+This command should be run regularly and you could use cron with the following settings to run the command every 15 minutes:
+
+```crontab
+*/15 * * * * /usr/bin/env php /path/to/datatidy/bin/console datatidy:data-flow:timeout-jobs > path/to/output.file
```

Comment:
Here `*/15*` sort of depends on the value of `$JOB_TIMEOUT_THRESHOLD`. It may make sense to add `timeout-threshold` parameter to the `datatidy:data-flow:timeout-jobs` command so we can make this dependency clearer, i.e. use something like

```
*/30 * * * * /usr/bin/env php /path/to/datatidy/bin/console datatidy:data-flow:timeout-jobs --timeout-threshold=30
```

to clean up jobs that have run for more than 30 minutes every 30 minutes.